cmd/emulator/start/start.go

```diff
@@ -42,7 +42,7 @@ type Config struct {
 	RestPort               int           `default:"8888" flag:"rest-port" info:"port to run the REST API"`
 	AdminPort              int           `default:"8080" flag:"admin-port" info:"port to run the admin API"`
 	DevWalletPort          int           `default:"8701" flag:"dev-wallet-port" info:"port to run Dev Wallet server"`
-	DevWalletEnabled       bool          `default:"false" flag:"dev-wallet" info:"enable local Dev Wallet server"`
+	DevWallet              bool          `default:"false" flag:"dev-wallet" info:"enable local Dev Wallet server"`
```

Comment:
Note 📖 

IMO, To be consistent with the flag `--dev-wallet` better to remove the `Enabled`.